common/src/main/java/io/homeassistant/companion/android/common/notifications/NotificationFunctions.kt

```diff
@@ -211,7 +214,7 @@ fun handleSmallIcon(
         val iconDrawable =
             IconicsDrawable(context, "cmd-$iconName")
         if (iconDrawable.icon != null) {
-            builder.setSmallIcon(iconDrawable.toAndroidIconCompat())
+            builder.setSmallIcon(iconDrawable.colorFilter { PorterDuffColorFilter(Color.WHITE, PorterDuff.Mode.SRC_IN) }.toAndroidIconCompat())

```

Comment:
Are we forcing the color to be white for android auto? Should we add a check so this only applies to android auto notifications to not impede on others?

Reply:
Yes it's forced. This shouldn't impede as for the status bar Android normally forces it to be white, but Android Auto doesn't for some reason which is why I'm adding it (MDI defaults to black).